db/migrate/20230614112328_add_user_ref_to_likes.rb

```diff
@@ -0,0 +1,7 @@
+class AddUserRefToLikes < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :likes, :user, null: false, foreign_key: true
```

Comment:
- [ ] When adding the `user` ref to the `Likes` table, it should be done through the `AuthorId` column in the `Likes` table based on this [ERD Diagram](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png). You have created a `user` ref but it should have `author_id` instead of `user_id` in the table. To handle this, I would suggest that you write your migration this way:

```ruby
def change
    add_reference :Likes, :author, null: false, foreign_key: { to_table: :users }
  end
```

Implementing it this way will map all Likes belonging to a user through the column `author_id`. Kindly implement this change. 👍 🍪